_posts/2022-11-07-changelog-182.markdown

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #182"
+date: 2022-11-07 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#8755] Support `let_chains` (by [@Kobzol])
```

Comment:
@Undin Should we provide a link to this RFC https://rust-lang.github.io/rfcs/2497-if-let-chains.html, despite the fact that it is now about the `let_chains_2` feature, not `let_chains`?

Reply:
I think yes